openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/courseware.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+  </head>
+
+  <body>
+    <iframe
+      id='unit-iframe'
+      src='{{ xblock_url }}'
+      allow='microphone *; camera *; midi *; geolocation *; encrypted-media *'
```

Comment:
Do you think that is necessary to set these values?

Reply:
@Squirrel18 They allow XBlocks to be able to interact with the parent window. I also added fullscreen since allowfullscreen is deprecated.